src/pg/db.js

```diff
@@ -19,24 +19,25 @@ import { put, patch } from './sql/index.js';
 import { format } from '@graffy/testing';
 import debug from 'debug';
 const log = debug('graffy:pg:dbRead');
-
+import { PgDb } from './db/pool';
 export class DbWrapper {
   // client is pgClient
-  constructor({ client, pgOptions }) {
-    this.client = client;
+  constructor({ pgOptions }) {
```

Comment:
Shouldn't it be `constructor(pgOptions)`? I don't expect any other options to be passed to pg module.